test/cmdlineTests.sh

```diff
@@ -133,37 +159,44 @@ function test_solc_behaviour()
     if [[ $exitCode -ne "$exit_code_expected" ]]
     then
         printError "Incorrect exit code. Expected $exit_code_expected but got $exitCode."
-        rm -f $stdout_path $stderr_path
         exit 1
     fi
 
     if [[ "$(cat $stdout_path)" != "${stdout_expected}" ]]
     then
         printError "Incorrect output on stdout received. Expected:"
-        echo -e "${stdout_expected}"
+        echo "${stdout_expected}"
 
         printError "But got:"
         cat $stdout_path
+
         printError "When running $SOLC ${filename} ${solc_args} <$solc_stdin"
```

Comment:
Not really part of this PR, but the output looks weird, if `$solc_stdin` is empty. Could you also add a condition for printing the additional `<`?

Reply:
Ok I changed how it print out the command